source/settings/plugin/mail-crypt-plugin.rst

```diff
@@ -1,31 +1,130 @@
 .. _plugin-mail-crypt:
 
-=====================
+=================
 mail-crypt plugin
-=====================
+=================
 
-The Mail-crypt-plugin is used to secure email messages stored in a Dovecot system. Messages are encrypted before written to storage and decrypted after reading. Both operations are transparent to the user.
+.. seealso:: See :ref:`mail_crypt_plugin` for an overview.
 
-In case of unauthorized access to the storage backend, the messages will, without access to the decryption keys, be unreadable to the offending party.
+Settings
+========
+
+.. _setting-plugin_mail_crypt_acl_require_secure_key_sharing:
+
+``mail_crypt_acl_require_secure_key_sharing``
+---------------------------------------------
+
+- Default: ``0``
+- Values:  :ref:`string`
+
+If true (``1`` or ``y``), require secure key sharing.
```

Comment:
I think just keep it as :ref:boolean. For some reason it reimplements it, but it's basically compatible.